Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
Opened #179 to change to HTTPS. If there's an issue with a developer machine related to HTTPS usage I'd be happy to assist.